src/content/docs/es/guides/server-side-rendering.mdx

```diff
@@ -147,7 +147,7 @@ Astro seguirá siendo un generador de sitios estáticos por defecto. Pero una ve
 
 ### Configuración de rutas individuales
 
-Ambos modos `server` e `hybrid` permiten páginas y endpoints renderizados en el servidor y renderizarán todas las páginas de acuerdo a su modo por defecto. Sin embargo, ambos modos permiten marcar una página individual para optar por no utilizar este comportamiento por defecto.
+Ambos modos `server` e `hybrid` permiten páginas y endpoints renderizados en el servidor y renderizarán todas las páginas de acuerdo a su comportamiento por defecto. Sin embargo, ambos modos permiten marcar una página individual para optar por no utilizar este comportamiento por defecto.
```

Comment:
Both options were a little bit confused, what do you think about this option?

Ambos modos `server` e `hybrid` permiten renderizar páginas y endpoints en el servidor y todas las páginas se renderizarán de acuerdo a su comportamiento por defecto. Sin embargo, ambos modos permiten marcar una página individual para no utilizar este comportamiento por defecto.